services/vision/builtin/tflite_test.go

```diff
@@ -254,3 +255,23 @@ func TestMoreClassifierModels(t *testing.T) {
 	test.That(t, bestClass[0].Label(), test.ShouldResemble, "292")
 	test.That(t, bestClass[0].Score(), test.ShouldBeGreaterThan, 0.93)
 }
+
+func TestInvalidLabels(t *testing.T) {
```

Comment:
Could you also add a test of it successfully unpacking a label file where all the labels are on one string, comma separated?